### PR TITLE
e2e:feat - certificate path

### PR DESCRIPTION
**Pull request by oliveirafelipezup**
**- What I did**
- New refactoring 
- Create tests for certificate path flag
- Create new file helper in testutils
**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: e2e/commands/start/start_test.go [hunk 1]
```diff
@@ -15,26 +15,35 @@
...
+		projectPath             = testutil.GoExample1
+		certificateFileWithPath string
 	)
 
+	BeforeSuite(func() {
+		file, err := os.CreateTemp(os.TempDir(), "*.crt")
+		if err != nil {
+			fmt.Print(err)
+		}
+		certificateFileWithPath = string(file.Name())
```
Comment (iancardosozup) [inline e2e/commands/start/start_test.go:44]: `file.Name()` already returns a string, there is no need to cast

**Review by iancardosozup — CHANGES_REQUESTED** [review]

File: e2e/commands/start/start_test.go [hunk 2]
```diff
@@ -15,26 +15,35 @@
...
+		flags                   map[string]string
+		repoAuthorization       string
+		projectPath             = testutil.GoExample1
+		certificateFileWithPath string
 	)
 
+	BeforeSuite(func() {
+		file, err := os.CreateTemp(os.TempDir(), "*.crt")
+		if err != nil {
+			fmt.Print(err)
```
Comment (matheusalcantarazup) [inline e2e/commands/start/start_test.go:42]: We should fail the test with some errors occurs.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: e2e/commands/start/start_test.go (hunk 1 above)
Comment (oliveirafelipezup) [reply e2e/commands/start/start_test.go]: adjusted

File: e2e/commands/start/start_test.go (hunk 2 above)
Comment (oliveirafelipezup) [reply e2e/commands/start/start_test.go]: adjusted!

File: e2e/commands/start/start_test.go [hunk 3]
```diff
@@ -17,28 +17,36 @@ package start_test
...
+		flags                   map[string]string
+		repoAuthorization       string
+		projectPath             = testutil.GoExample1
+		certificateFileWithPath string
 	)
 
+	BeforeSuite(func() {
+		file, err := os.CreateTemp(os.TempDir(), "*.crt")
+		if err != nil {
+			ginkgo.Fail(fmt.Sprintf("error: %v", err))
```
Comment (matheusalcantarazup) [inline e2e/commands/start/start_test.go:45]: Use just `Fail(fmt.Sprintf("error: %v", err))`
ginko is already imported using `.`

File: internal/utils/testutil/file.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	"strings"
+)
+
+func CreateTempFileCrt() string {
+	file := os.NewFile(0, "temp.crt")
+
+	newLocation := os.TempDir() + file.Name()
+
+	return strings.ReplaceAll(newLocation, `\`, `\\`)
+}
```
Comment (matheusalcantarazup) [inline internal/utils/testutil/file.go:28]: This function seems not used. If it's not, please remove this file.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: e2e/commands/start/start_test.go (hunk 3 above)
Comment (oliveirafelipezup) [reply e2e/commands/start/start_test.go]: adjusted!

File: internal/utils/testutil/file.go (hunk 4 above)
Comment (oliveirafelipezup) [reply internal/utils/testutil/file.go]: adjusted

File: internal/utils/testutil/examples.go [hunk 5]
```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	// AverageTimeoutAnalyzeForExamplesFolder Average timeout for analysis in the example directory
+	// Average timeout for analysis in the example directory
```
Comment (matheusalcantarazup) [inline internal/utils/testutil/examples.go:26]: This change seems not necessary.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: e2e/commands/start/start_test.go [hunk 6]
```diff
@@ -61,11 +69,11 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 	})
 
 	When("global flag --config-file-path is passed", func() {
-		var configFilePathToTest = filepath.Join(configFilePath, "horusec-config.json")
+		var configFilePathToTest = filepath.Join(projectPath, "horusec-config.json")
```
Comment (matheusalcantarazup) [inline e2e/commands/start/start_test.go:72]: Use short variable declaration `configFilePathToTest := filepath.Join(projectPath, "horusec-config.json")`

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

**Review by matheusalcantarazup — APPROVED** [review]

**Review by iancardosozup — APPROVED** [review]

**Pull request merged**